### ISSUE TITLE
IS_LOCAL env var not being set on local

### ISSUE THREAD

**Issue by bernardobelchior** [softprops/serverless-rust#34]
## 🐛 Bug description

According to the [`AWS - Invoke Local`](https://serverless.com/framework/docs/providers/aws/cli-reference/invoke-local/) page in the Serverless documentation, running `sls invoke local` should set the `IS_LOCAL` environment variable.
I have tried with a NodeJS lambda function and the variable is set, which leads me to believe that the problem comes from this package.

#### 🤔 Expected Behavior

The `IS_LOCAL` environment variable should be set.

#### 👟 Steps to reproduce

Having the serverless framework installed:

```
git clone https://github.com/bernardobelchior/serverless-report
cd serverless-report
npm install
sls invoke local -f hello
```

#### 🌍 Your environment

serverless version: 1.40.0

rust-plugin version: 0.3.2

OS: Linux

**Comment by softprops** [comment]
interesting. I'll take a look.

**Comment by softprops** [comment]
noting this for reference and follow up https://github.com/serverless/serverless/pull/5988

**Comment by bernardobelchior** [comment]
I did face the same issue. I couldn't pass environment variables like that either, so it may be a problem with the framework itself.

**Comment by bernardobelchior** [comment]
https://github.com/serverless/serverless/pull/5988 was merged. 
[Fix included in v1.41.0](https://github.com/serverless/serverless/pull/6025) of serverless, which should be out shortly.
